files/zh-cn/web/css/css_flow_layout/flow_layout_and_writing_modes/index.md

```diff
@@ -31,7 +31,7 @@ CSS 编写模式级别 3 规范定义了文档编写模式更改对流布局的
 
 {{EmbedGHLiveSample("css-examples/flow/writing-modes/horizontal-tb.html", '100%', 720)}}
 
-`horizontal-tb` 值为你提供了一个从右到左的块流方向和一个垂直的内联方向，如下一个示例所示。
+`vertical-rl` 值为你提供了一个从右到左的块流方向和一个垂直的内联方向，如下一个示例所示。
```

Comment:
Let's update translation for 'block flow direction' and 'inline direction'

ref: https://github.com/mdn/translated-content/issues/12632

```suggestion
`vertical-rl` 值为你提供了一个从右到左的块向和一个垂直的行向，如下一个示例所示。
```